_studio/mfx_lib/encode_hw/shared/ehw_device_vaapi.cpp

```diff
@@ -339,16 +339,21 @@ mfxStatus DeviceVAAPI::Init(
 mfxStatus DeviceVAAPI::QueryStatus(DDIFeedback& ddiFB, mfxU32 id)
 {
     MFX_CHECK(!ddiFB.Get(id), MFX_ERR_NONE);
+    mfxStatus sts;
```

Comment:
nit: I'd rather left declaration on both lines: 348 and 355

Reply:
Done